### DISCUSSION TITLE
Current position of object

### DISCUSSION THREAD

**Discussion by sander45** [blakeblackshear/frigate#3220] [Ask A Question]
Is there a way to get live information on the location of an object that has been recognized?

From the docs: A MQTT message with the location is sent when the tracked object is no longer marked as a false positive and when the end of event. And in some occasions in between. 
However I want to track the objects location the whole time.

I want to use this knowledge to remind my dog with an aimed water beam that she should not pee on the gras. Maybe it is an option to define say 50 zones, so that frigate will send an MQTT update when ze moves to a different zone.  

Any other suggestions?

Thanks.

PS
It remains the question whether the system is fast enough. I have Frigate currently running with docker on a separate computer. The Raspberry pi hosts Home Assistant. I have a Wemos that deals with all garden sprayers.

**Comment by blakeblackshear** [comment]
The events topic publishes the bounding box as "box". https://docs.frigate.video/integrations/mqtt

**Comment by sander45** [comment]
Thanks, but I think this is not updated for each frame/or second. It only the position when the dog becomes a false positive, but not the location afterwards. 
Can I force a MQTT event update, so can poll her current location and track her movements?
Thanks again

**Reply by NickM-27** [reply]
> Thanks, but I think this is not updated for each frame/or second. It only the position when the dog becomes a false positive, but not the location afterwards. 

I think you read that wrong:

**The first message is published when the tracked object is no longer marked as a false_positive.**

**When frigate finds a better snapshot of the tracked object or when a zone change occurs, it will publish a message with the same id. When the event ends, a final message is published with end_time set.**

Not quite every second but any zone changes will trigger an update

**Comment by sander45** [comment]
Thanks. So I think the best option is to define serval dozen zone's to get a more accurate update.
